{
  "Header": {
    "Original": "CVE-2016-1000212 [Mitigation for HTTPoxy vulnerability]",
    "Line": 437172,
    "ID": "CVE-2016-1000212",
    "Description": "[Mitigation for HTTPoxy vulnerability]"
  },
  "Annotations": [
    {
      "Original": "{DSA-3642-1 DLA-583-1}",
      "Line": 437173,
      "Type": "xref",
      "Bugs": [
        "DSA-3642-1",
        "DLA-583-1"
      ]
    },
    {
      "Original": "- lighttpd 1.4.43-1 (bug #832571)",
      "Line": 437174,
      "Type": "package",
      "Package": "lighttpd",
      "Kind": "fixed",
      "Version": "1.4.43-1",
      "BugNo": 832571
    },
    {
      "Original": "NOTE: https://redmine.lighttpd.net/projects/lighttpd/repository/revisions/779c133c16f9af168b004dce7a2a64f16c1cb3a4/diff",
      "Line": 437175,
      "Type": "NOTE",
      "Description": "https://redmine.lighttpd.net/projects/lighttpd/repository/revisions/779c133c16f9af168b004dce7a2a64f16c1cb3a4/diff"
    },
    {
      "Original": "NOTE: CVE assigned for the mitigation to identify the fix. But it is not a vulnerability in lighttpd itself.",
      "Line": 437176,
      "Type": "NOTE",
      "Description": "CVE assigned for the mitigation to identify the fix. But it is not a vulnerability in lighttpd itself."
    }
  ]
}